doc/jenkins.rst

```diff
@@ -0,0 +1,136 @@
+    This means that without changing server settings, interactive features of the generated log will be
+    inaccessible when opened directly from Jenkins artifacts.
+
+
+Here's the list of steps we performed to integrate the interactive log with Jenkins server:
+
+1. :ref:`Add a command line option <jenkins#add_arg>` to generate a self-contained HTML file
+2. :ref:`Add a Jenkins plugin <jenkins#plugin>` to integrate a generated log into a Jenkins job
+3. :ref:`Set up Resource Root URL <jenkins#resource_url>` to allow Jenkins rendering interactive content
+4. :ref:`Configure reverse proxy <jenkins#nginx>` to handle multiple domains interaction
+
+
+.. _jenkins#add_arg:
+
+Add command line option
+-----------------------
+
+To generate a single interactive self-contained HTML file, pass an `--html-log <args.html#Output>`__
+option to command line. It will be stored to in project `artifacts <args.html#Artifact\ collection>`__ folder.
+Note that the log name can either be specified or left default.
+
+.. note::
+
+    Jenkins jobs do not show any files via web interface if not not specifically configured to do so. Use
+    ``archiveArtifacts`` in Pipeline or ``Archive the artifacts`` in `Post-build Actions` to check the file presence
+    and contents before next step if needed
+
+
+.. _jenkins#plugin:
+
+Add a Jenkins plugin
+--------------------
+
+A `HTML Publisher <https://plugins.jenkins.io/htmlpublisher/>`__ plugin is a very convenient way to add an HTML report
+to a Jenkins job. To use it, you will need to:
+
+1. Install it on server (server restart might be needed to apply changes)
+2. Add ``-hl``/``--html-log``/``-hl <name>`` option to the `Universum` command line as mentioned in previous section
+3. Pass the log name to plugin configuration as described in manual (https://plugins.jenkins.io/htmlpublisher/)
+   via `Post-build Actions` or Pipeline
+4. Launch a configured job at least once for the log to be generated
+5. Let Jenkins render the interactive content of log (see the next section)
+
+
+.. _jenkins#resource_url:
+
+Set up Resource Root URL
+------------------------
+
+As already mentioned above, due to `Jenkins Content-Security-Policy
+<https://www.jenkins.io/doc/book/security/configuring-content-security-policy/>`__ some features of an interactive log
+might not work properly, and its contents might be not shown or displayed incorrectly.
+
+A recommended way to allow Jenkins server to render interactive user content is to `configure Jenkins to publish
+artifacts on a different domain <https://www.jenkins.io/doc/book/security/user-content/#resource-root-url>`__
+by changing ``Resource Root URL`` in `Manage Jenkins » Configure System » Serve resource files from another domain`
+from something like ``my.jenkins.io`` to ``res.my.jenkins.io``.
+
+Note that Jenkins interaction with resource domain, resolved to the same host IP is not done via ``localhost``
```

Comment:
I think you can mention that jenkins requires some initial interaction with resource domain during changing of this setting.